pop_up.js

```diff
@@ -0,0 +1,262 @@
+    image: 'images/tonicSmall.svg',
+    image2: 'images/tonic4.svg',
+    description: `  A smart assistant to make driving more safe, efficient, and fun by
+                            unlocking your most expensive computer: your car.`,
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['UBER', 'Lead Developer', '2018'],
+  },
+];
+
+const portfolioArticle = document.getElementById('portfolio');
+portfolioArticle.innerHTML = projectCards.map((project, index) => `
+  <article class="card ${index % 2 !== 0 ? 'card--two' : 'card-one'}">
+        <img class="card__image" src="${project.image}" alt="Tonic image" />
+        <img
+          class="card__image2 image-scale"
+          src="${project.image2}"
+          alt="Nature image"
+        />
+        <h2 class="card__title">${project.name}</h2>
+        <div class="card-items">
+          <h2 class="title__desk">${project.name2}</h2>
+          <div class="card__subtitle">
+            <h3>${project.specifics[0]}</h3>
+            <img src="images/counter.svg" alt="Counter icon" />
+            <h3 class="light-text">${project.specifics[1]}</h3>
+            <img src="./images/counter.svg" alt="Counter icon" />
+            <h3 class="light-text">${project.specifics[2]}</h3>
+          </div>
+          <div class="subtitle__desk">
+            <h3>${project.specifics2[0]}</h3>
+            <img src="./images/counter.svg" alt="Counter icon" />
+            <h3 class="light-text">${project.specifics2[1]}</h3>
+            <img src="./images/counter.svg" alt="Counter icon" />
+            <h3 class="light-text">${project.specifics2[2]}</h3>
+          </div>
+          <div class="card__body">
+            <p class="para-mobile">
+            ${project.description}
+            </p>
+            <p class="para-desk move-text">
+             ${project.description}
+            </p>
+            <ul class="categories">
+              <li>${project.technologies[0]}</li>
+              <li>${project.technologies[1]}</li>
+              <li>${project.technologies[2]}</li>
+            </ul>
+            <ul class="categories-desk">
+               <li>${project.technologies[0]}</li>
+              <li>${project.technologies[1]}</li>
+              <li>${project.technologies[2]}</li>
+            </ul>
+            <button data-modal="#modal" class="btn" type="button">
+              See Project
+            </button>
+          </div>
+        </div>
+ </article>
+  `).join('');
+
+/* Modal Window */
+const openModelbtn = document.querySelectorAll('[data-modal]');
+const closeModelbtn = document.querySelectorAll('[data-close-button]');
+const overlay = document.getElementById('overlay');
+const titleModal = document.querySelector('.title');
+const imageModal = document.querySelector('.modal-img');
+const descriptionModal = document.querySelector('.modal-desc p');
+const categoriesModal = document.querySelector('.modal-categories');
```

Comment:
- [ ] Add more padding to modal categories in order to better match the design:
Yours:
![image](https://user-images.githubusercontent.com/62705947/217581064-940c6d79-781c-4b33-8e32-22791231cf76.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/217581125-0d5ad50a-eeb7-4549-87ba-79a9205f3f8b.png)

Reply:
- [ ] Your modal on mobile is flicking, kindly fixi t
https://www.loom.com/share/2c5dffe3b1554474b3f29044d5b4f6a7

Reply:
- [ ] Kindly remove some of this empty space since it is not matching the design(it is a lot bigger than on design)
![image](https://user-images.githubusercontent.com/62705947/217582277-a56ce4dc-bcca-4fa2-8cc1-b681cd48f765.png)

Reply:
- [ ] On the readme file kindly remove the FAQ since it is optional. Or you can add real questions and answers